src/Exception/OperationTimedOut.php

```diff
@@ -13,4 +13,20 @@
 
 class OperationTimedOut extends \Exception
 {
+    public static function createFromTimeout(int $timeoutMicroSec): self
+    {
+        return new self('Operation timed out ('.self::getTimeoutPhrase($timeoutMicroSec).')');
+    }
+
+    private static function getTimeoutPhrase(int $timeoutMicroSec): string
```

Comment:
return type is incorrect, and only not crashing because strict types is not enabled. ;)